frontend/src/components/common/Button.vue

```diff
@@ -66,7 +66,11 @@ function onClicked(event: any) {
     </Button>
 </template>
 <style lang="scss">
-
+.nr-button {
+    display: flex;
```

Comment:
Hi Nick, this line we already removed in Jota's last pr, we don't need it anymore.

Reply:
This has been removed. Thanks.